.codegen/__init__.py.tmpl

```diff
@@ -34,6 +35,7 @@ class WorkspaceClient:
         self.config = config
         self.dbutils = dbutils.RemoteDbUtils(self.config)
         self.api_client = client.ApiClient(self.config)
+        self.files = FilesMixin(self.api_client)
```

Comment:
Does this need to be added here explicitly because there is no Files tag in the OpenAPI spec?

Reply:
yes, temporarily